package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@perimetre/ui",
   "description": "A component library made by @perimetre",
-  "version": "9.3.1",
+  "version": "9.3.2",
```

Comment:
Wouldn't it be a minor version instead of a bugfix?

Reply:
I consider this a fix because we're fixing the modal variant `new` to conform with the new designs. A couple of weeks back I had already fixed that variant but it was still not as the final designs.